rna/rna-index/rna-index.inputs.json

```diff
@@ -0,0 +1,16 @@
+{
+  "##_COMMENT1": "Set your per-sample values: name and BAM path",
+  "RNAIndex.sample_name": "__INSERT_SAMPLE_NAME__",
+  "RNAIndex.star_bam_file":"__INSERT_GCS_PATH_TO_BAM_FILE__",
+
+  "##_COMMENT2": "Path to a Docker image containing samtools",
+  "RNAIndex.samtools_docker":"biocontainers/samtools:v1.7.0_cv4",
+
+  "##_COMMENT3": "Would be unusual to take more than 24 hours, so use preemptible",
+  "RNAIndex.preemptible_tries": 5,
+
+  "RNAIndex.num_cpu_cores": 4,
```

Comment:
We ultimately used much more modest VM resources, which went into the 
[Controlling Cloud costs - sample use cases](https://support.terra.bio/hc/en-us/articles/360029772212-Controlling-Cloud-costs-sample-use-cases) Terra article:

> We found that we could index BAMs on a small, single-core VM (n1-standard-1) with a 200 GB disk.
> At preemptible rates of $0.01 per hour for the VM and $0.01 for the disk, these workflows cost pennies
> to complete for each sample.

Please update these inputs to reflect that machine shape.